asynction/types.py

```diff
@@ -12,12 +14,259 @@
 from svarog import forge
 from svarog import register_forge
 from svarog.types import Forge
+from typing_extensions import TypedDict
+
+GLOBAL_NAMESPACE = "/"
 
 JSONMappingValue = Any
 JSONMapping = Mapping[str, JSONMappingValue]
 JSONSchema = JSONMapping
 
-GLOBAL_NAMESPACE = "/"
+
+class SecurityInfo(TypedDict):
+    """Security handler function response type.
+
+    One of scopes, scope and one of sub, uid must be present
+
+    Subclass this type to add extra fields to a security handler resoponse
+    """
+
+    scopes: Sequence[str]
+    scope: str
+    sub: Any
+    uid: Any
+
```

Comment:
This type looks good, thanks! Probably fits better under `security.py` I think.
Also would be good to expose it in `asynction/__init__.py` so that it can be imported by users through: `from asynction import SecurityInfo.`